src/main/java/ch/uzh/ifi/hase/soprafs22/controller/ChatController.java

```diff
@@ -86,7 +102,7 @@ public boolean hasUnreadMessages(@RequestHeader(value = "authorization", require
     return false;
   }
 
-
+  // TODO: instead create chat when match is created?
   @PostMapping("/users/{userId}/chats")
   @ResponseStatus(HttpStatus.CREATED)
   @ResponseBody
```

Comment:
We don't need this postmapping anymore, but can just create a chat entity with the match entity. (Advise)

Reply:
already done, i delte this postMapping